variables.tf

```diff
@@ -321,12 +249,12 @@ variable "master_ipv4_cidr_block" {
 
 variable "release_channel" {
   type        = string
-  description = "(Optional) The release channel of this cluster. Accepted values are `UNSPECIFIED`, `RAPID`, `REGULAR` and `STABLE`. Defaults to `UNSPECIFIED`."
-  default     = "UNSPECIFIED"
+  description = "(Optional) The release channel of this cluster. Accepted values are `UNSPECIFIED`, `RAPID`, `REGULAR` and `STABLE`."
+  default     = "STABLE"
```

Comment:
why remove unspecified? 

-> you forgot to remove it also in description.

Reply:
because unspecified is not supported in autopilot

Reply:
fixed in bcb11652